package.json

```diff
@@ -125,7 +125,12 @@
 	},
 	"dependencies": {},
 	"lint-staged": {
-		"*.{md,json,js,ts}": [
+		"*.{md,json,ts}": [
+			"prettier --write",
+			"git add"
+		],
+		"*.{js}": [
+			"npm run eslint --fix",
```

Comment:
Thanks for adding `--fix`, but this is still incorrect (for the same reason). We do not want to lint the _entire_ source tree just because a single file was `git add`ed.

Please replace this with:

```
"*.js": [
  "eslint --fix",
  "prettier --write",
  "git add"
]
```